framework/src/auxkernels/AdvectiveFluxAux.C

```diff
@@ -16,7 +16,7 @@ InputParameters
 AdvectiveFluxAux::validParams()
 {
   InputParameters params = AuxKernel::validParams();
-  MooseEnum component("x y z normal");
+  MooseEnum component("x y z normal", "x");
```

Comment:
```suggestion
  MooseEnum component("x y z normal");
```
no default is better here, users could forget the parameter